### PR TITLE
SPOI-10927 Example app s3 tuple output module

### PR DESCRIPTION
**Pull request by yogidevendra**

### CODE REVIEW

File: tutorials/s3-tuple-output/src/main/java/com/datatorrent/tutorial/s3input/Application.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * Simple application illustrating file copy from S3
+ */
+@ApplicationAnnotation(name="s3-output-line")
+public class Application implements StreamingApplication
+{
+
+  public void populateDAG(DAG dag, Configuration conf)
+  {
+    //LineByLineFileInputOerator lineInputOperator = dag.addOperator("lineInput", LineByLineFileInputOperator.class);
```
Comment (chaithu14) [inline tutorials/s3-tuple-output/src/main/java/com/datatorrent/tutorial/s3input/Application.java]: Remove these lines.

**Review by chaithu14 — COMMENTED** [review]
README file is missed and add the details of this application in tutorials/README.md file.

File: tutorials/s3-tuple-output/src/main/resources/META-INF/properties.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  </property>
+  <property>
+    <name>dt.operator.lineInput.prop.minReaders</name>
+     <value>1</value>
+  </property>
+  <property>
+    <name>dt.loggers.level</name>
+    <value>org.apache.apex.malhar.lib.fs.s3.*:DEBUG,org.apache.apex.*:DEBUG,com.datatorrent.stram.plan.physical.*:DEBUG,com.datatorrent.lib.*:DEBUG</value>
+  </property>
+</configuration>
```
Comment (chaithu14) [inline tutorials/s3-tuple-output/src/main/resources/META-INF/properties.xml]: Add new line

Comment (yogidevendra) [inline tutorials/s3-tuple-output/src/main/resources/META-INF/properties.xml]: done

**Pull request merged**